evap/staff/views.py

```diff
@@ -1512,9 +1516,12 @@ def evaluation_textanswers_skip(request):
 @require_POST
 @reviewer_required
 def evaluation_textanswers_update_publish(request):
-    textanswer_id = request.POST["id"]
-    action = request.POST["action"]
-    evaluation_id = request.POST["evaluation_id"]
+    try:
+        textanswer_id = request.POST["id"]
+        action = request.POST["action"]
+        evaluation_id = request.POST["evaluation_id"]
```

Comment:
Same here, depending on whether we want the logging or not, by replacing all of these with `dict.get` and using `get_object_or_404` below, we will produce 400s if any of these are empty. Don't know if we want to distinguish between 400 and 404 here either